lib/matplotlib/mlab.py

```diff
@@ -152,12 +152,12 @@ def detrend(x, key=None, axis=None):
         Array or sequence containing the data.
 
     key : {'default', 'constant', 'mean', 'linear', 'none'} or function
-        Specifies the detrend algorithm to use. 'default' is 'mean', which is
-        the same as `detrend_mean`. 'constant' is the same. 'linear' is
-        the same as `detrend_linear`. 'none' is the same as
-        `detrend_none`. The default is 'mean'. See the corresponding
-        functions for more details regarding the algorithms. Can also be a
-        function that carries out the detrend operation.
+        The detrending algorithm to use. 'default' is 'mean', which is the
+        same as `detrend_mean`. 'constant' is the same. 'linear' is the same
```

Comment:
missing what `constant` is the same as. 
Also, are these equivalent keywords: 
`linear` and `detrend_linear` or is the latter the algorithm specified by the former?

Reply:
reworded for constant.  passing 'linear' is the same as passing `detrend_linear`.